server/neptune/temporalworker/server.go

```diff
@@ -191,3 +202,46 @@ func Healthz(w http.ResponseWriter, _ *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	w.Write(data) // nolint: errcheck
 }
+
+func createJobsController(config *neptune.Config, jobOutputHandler *job.OutputHandler) *controllers.JobsController {
+	projectJobsScope := config.Scope.SubScope("getprojectjobs")
```

Comment:
```suggestion
	projectJobsScope := config.Scope.SubScope("http.getjob")
```

feel like we can emit project going forward, and namespacing by http just makes it clear that it's an api call.